src/pkg/intentsoutput/intentslister/lister.go

```diff
@@ -21,3 +22,28 @@ func ListFormattedIntents(intents []v1alpha2.ClientIntents) {
 		}
 	}
 }
+
+type ListedIntent struct {
+	Name      string            `json:"name"`
+	Namespace string            `json:"namespace"`
+	Calls     []v1alpha2.Intent `json:"calls"`
+}
+
+func ListFormattedIntentsJSON(intents []v1alpha2.ClientIntents) error {
```

Comment:
Let's create one function who recive the format and data, prints the data as json or sends the json object to 
`ListFormattedIntents` and make this function work with template, so we have one function `ListFormattedIntents(data, format)`

template := `
{{- range . }}
{{ .Name }} in namespace {{ .Namespace }} calls:
{{- range .Calls }}
	  - {{ .ServerName }} in namespace {{ .ServerNamespace }}
{{- range .Topics }}
	    - Kafka topic: {{ .Name }}, operations: {{ .Operations }}
{{- end }}
{{- range .HTTPResources }}
	    - path {{ .Path }}, methods: {{ .Methods }}
{{- end }}
{{- end }}
{{- end }}
{{- end }}
`